rock.osdeps

```diff
@@ -235,7 +235,7 @@ dc1394:
 
 vtk:
     debian,ubuntu:
-        '14.04,14.10,15.04,15.10':
+        '14.04,14.10,15.04,15.10,16.04':
```

Comment:
libvtk5-qt4-dev just became unavailable in debian testing.
https://github.com/rock-core/package_set/pull/90/files

Afaik the dependency that adds vtk-qt is the rock-widget-collection, i don't mind changing the rock.core osdep because i just fixed the vtk-qt4 one 10 days ago:

https://github.com/rock-core/package_set/pull/84/files